test/features/steps/step_implementation.py

```diff
@@ -122,6 +122,14 @@ def step_impl(context):
     context.logger.info("\n{}".format(context.stderr))
 
 
+@then('No exceptions')
```

Comment:
It is better to name this as `No tracebacks`, as exception `ValueError` is widely used as a method to abort the process without printing backtrace.